cmd/kubeadm/app/cmd/phases/init/waitcontrolplane.go

```diff
@@ -47,7 +47,13 @@ var (
 		- 'journalctl -xeu kubelet'
 
 	Additionally, a control plane component may have crashed or exited when started by the container runtime.
-	To troubleshoot, list all containers using your preferred container runtimes CLI, e.g. docker.
+	To troubleshoot, list all containers using your preferred container runtimes CLI.
+
+	Here is one example how you may list all Kubernetes containers running in cri-o/containerd:
```

Comment:
Ok, this may be confusing a bit to users. In essence we need to display only stuff that is relevant to the use case at hand.
Hence we should be doing:
```
{{ if .IsDocker }}
 display docker stuff
{{ else }}
 display crictl stuff
{{ end }}
```

You can display the contents of `CRISocket` in place of `RUNTIME_SOCKET`. You can use `NodeRegistration.CRISocket == constants.DefaultDockerCRISocket` to detect if you are dealing with Docker or not.
`CRISocket` won't be deprecated until we decide to move away from CRI support which is unlikely to happen in the foreseeable future. It's just that we don't want to dissect its values.

Reply:
agh, So I just need to revert to what it was before which did exactly that :)

I panicked thinking that using the CRISocket was gonna be deprecated. Let me restore the code to what it was before.

Thanks for the pointers!

Reply:
With a slight modification to reversing the `if` statement and using a `isDocker` boolean it was better.

Reply:
problem with that is that the `ctx` type map that we are passing to the template is `string` so I canty directly pass a bool in there, can I?

Reply:
You can convert it to a struct if that fits better.

Reply:
done!